pypeit/par/pypeitpar.py

```diff
@@ -686,13 +686,9 @@ def __init__(self, spec_method=None, spec_maxshift=None, spectrum=None):
 
         # Fill out parameter specifications.  Only the values that are
         # *not* None (i.e., the ones that are defined) need to be set
-        defaults['spec_method'] = 'skip'
-        options['spec_method'] = FlexurePar.valid_methods()
-        dtypes['spec_method'] = str
-        descr['spec_method'] = 'Method used to correct for flexure. Use skip for no correction.  If ' \
-                          'slitcen is used, the flexure correction is performed before the ' \
-                          'extraction of objects (not recommended).  ' \
-                          'Options are: None, {0}'.format(', '.join(options['spec_method']))
+        defaults['perform'] = False
```

Comment:
Can we call this `spec_perform`? To be clear it is for spectral flexure.

Reply:
Agreed.